data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			choice
+				`	"I didn't expect you to be back to working so soon."`
+					goto working
+				`	"I didn't think you would be trusted enough to have access to Hai technology already."`
+				`	"I wasn't sure you would help the Hai at all."`
+					goto help
+			`	They make a weird sound that's probably laughter. "You call this 'Mesuk,'" he says, showing you the cooler. "Even our 'Rem' 'Sorobeh' can do something so primitive but to speak 'Ter' tongue there are some interesting 'Estar' here 'Erma' not made by 'Unkor' races like you and the Hai like the 'E Shema' cooler they stole from 'Ter' 'Unkor' will never 'Tema'."`
+				goto objective
+			label help
+			`	You seem to have hit a nerve. "'Ter'... Have a 'Noc' toward the Unfettered but needs to be 'Terma' else 'Ter' cannot be free then they will have a 'Noc' with 'Ter.'"`
+				goto objective
+			label working
+			`	"Unlike it 'Ter' are 'Akeh' 'Ter' always stay 'Akeh' even when having to 'Keh' what you call foood it is 'Eta' nutrigel."`
+			choice
+				`	"I see why the Hai allowed you to access their technology .You think in a similar way."`
+				`	"Oh, is that why you got captured by the Alphas?"`
+			`	They seem to grumble a bit about what you just said, but the translator cannot decipher any meaning. Considering what they could be saying, it's probably for the best.`			
+			label objective
+			`	Taer-Fi stops whatever it is that she was doing on the cooler and comes to speak to you. "We have been working on improved cooling systems for a while now. To speed it up we could use some samples of other 'primitive' cooling technologies to work on as base materials. I thought I would ask you, considering you came from a very... 'young' civilization," she explains.`
+			choice
+				`	"That can't be hard. I'll go look for human coolers then."`
+					goto quantity
+				`	"Can you be more specific?"`
+				`	"I'm not interested in helping you any more."`
+					decline
+			`	"Not really. I don't know all kinds of cooling your species could have, but it most be large, considering your technology is not as heat efficient as ours. You're probably even using things like water, ha! Don't bring us something quite that primitive. We want the good stuff, like your nitrogen cooling, for instance. Do bring a lot of it, say, 300 tons."`
+			choice
+				`	"I'll go look for those then."`
+					accept
+				`	"I'm not interested in helping you any more."`
+					decline
+			label quantity
+			`	"Be sure to bring back 300 tons worth of nitrogen coolers so we have a solid base to work on. Don't bring us things like simple, water-based systems."`
+				accept
+
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on visit
+		dialog `You do not yet have 300 tons of nitrogen human coolers that Taer-Fi requested in your cargo hold or installed on your ships. Unselect your ship with "ctrl+click to load in the cargo hold".`
```

Comment:
Check the existing tutorial `dialog`s to make sure the formatting lines up with those.

Reply:
There doesn't seem to be much to those in terms of formatting, but I changed this a bit to match